index-handlers/src/main/java/no/sikt/nva/nvi/index/IndexNviCandidateHandler.java

```diff
@@ -20,14 +20,14 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class IndexNviCandidateHandler implements RequestHandler<SQSEvent, Void> {
+public class IndexNviCandidateHandler implements RequestHandler<DynamodbEvent, Void> {
```

Comment:
I think this should now be called UpdateIndexHandler or something like this.

Reply:
Definitely change the name since you're checking if it is a Insert/Modify/Delete, but do this in a new PR that covers just this. Naming-wise, it is probably clearest to use `UpdateIndexDocumentHandler`